stac-api/src/main/java/io/arlas/server/stac/api/StacRESTService.java

```diff
@@ -271,6 +263,7 @@ protected StacFeatureCollection getStacFeatureCollection(CollectionReference col
             Arrays.asList("self", "next", "previous").forEach(rel -> {
                 if (context.containsKey(rel)) {
                     if (method.equals("POST")) {
+                        // TODO : fix when body is null!!
```

Comment:
needs to create an issue ?

Reply:
https://github.com/gisaia/ARLAS-server/issues/948